MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -195,6 +193,12 @@ public void onAdsManagerLoaded(AdsManagerLoadedEvent adsManagerLoadedEvent) {
     }
   }
 
+  public void allowHeaderToBeSentToBackend(String headerName) {
```

Comment:
We don't want this to be public, I don't think. This should be protected/package access, right? As in, we don't want customers, today, to be able to whitelist any header they want back to us. I think you can make it `package` and still access it in the automated tests/tests that you need, right? Either way, maybe we can use proguard to make sure this isn't available, but I'm pretty sure we don't want this publicly added to our API (which I think this does).

Reply:
I think that as part of the specification we should allow user to whitelist the header. It say like this: "We should only capture x-cdn and content-type headers (but that list should be updatable via a code change)
", what are your thoughts

Reply:
First try making this `private`, else default scope
```suggestion
  private void allowHeaderToBeSentToBackend(String headerName) {
```